packages/js-dpp/lib/identity/IdentityPublicKey.js

```diff
@@ -240,6 +240,7 @@ IdentityPublicKey.PURPOSES = {
   AUTHENTICATION: 0,
   ENCRYPTION: 1,
   DECRYPTION: 2,
+  WITHDRAW: 3,
```

Comment:
The implementation here doesn't align with the Identity DIP: https://github.com/dashpay/dips/blob/master/dip-0011.md#identity-public-key which uses purpose `3` for something else and doesn't include withdraw. That difference will need to be reconciled with the DIP at some point.